index.html

```diff
@@ -31,24 +31,28 @@
             <li class="menu-icon">
               <img src="./images/icons/Icon-Menu.svg" alt="menu" />
             </li>
+            <li class="menu-icon display-none">
+              <img src="./images/icons/Cancel.svg" alt="menu" />
```

Comment:
Please ensure you add `the close (X) button` to the menu display after a user clicks the `hamburger button`. So the user can click (or tap) the close (X) button, to make the mobile menu disappear according to the project requirement.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/156064808-490e352f-f27f-4e51-b078-5146cf4fb329.png) | ![image](https://user-images.githubusercontent.com/10332499/156064755-e947abf0-a3bb-46cf-8b16-3629d4899dc0.png)

Reply:
@micheaol thank you for your review and requesting changes. I will do that now.